Lab 2 Changes (#252)
aws-samples/aws-hpc-tutorials: content/04-container-parallelcluster/06-delete-pc.md

+++
title = "e. Terminate Your Cluster"
date = 2019-09-18T10:46:30-04:00
weight = 60
tags = ["tutorial", "delete", "ParallelCluster"]
+++

Now that you are done with your HPC cluster, you can delete it.

> Delete the cluster by clicking **Delete** from the pcluster manager console and clicking confirm.

Reviewer: ```suggestion
From the **pcluster manager** console, choose **Delete** and confirm to start the cluster deletion
```

![Delete Cluster](/images/container-pc/delete.png)

> The cluster and all its resources will be deleted by CloudFormation. You can check the status on the **Stack Events** tab.

Reviewer: ```suggestion
The cluster and all its resources will be deleted by AWS CloudFormation. You can check the status on the **Stack Events** tab.
```

> The cluster and all its resources will be deleted by CloudFormation. You can check the status on the **Stack Events** tab.

Reviewer: If we are removing cloudformation info in lab1 we should not talked about cloudformation.
